src/app/(main)/EventPage.tsx

```diff
@@ -51,8 +52,20 @@ const Roster = ({participants, orgs, startTime}: {participants: Record<string, P
       const isToday = new Date().setHours(0,0,0,0) === new Date(o.value).setHours(0,0,0,0);
       return `${!isToday ? formatDate(o.value, 'yyyy-MM-dd ') : ''}${formatDate(o.value, 'HHmm')}`;
     }, flex: 1 },
+    { field: 'eta', headerName: 'ETA', flex: 1},

```

Comment:
This won't show a date, so it will be different from the "Time" column when the eta isn't today. Maybe that's ok, since this is only useful for a short time.

We could consider rendering it as a relative time, too, like "85m", or "-5m" if someone is overdue. Depends on which you think would be more useful.

Reply:
Yeah, I think you might need to revisit this in the future, especially if there's ETAs that span over midnight, it will always assume the time is today. So if the mission spins up at 2300 and ETA is 0030 it will think the ETA is in the past. That's why I'm only rendering the time, for now. Maybe we add some logic to force the date to plus one If it's otherwise in the past, and then we could do relative time calculations. Which actually might be hopeful. Or maybe an option where it toggles.

Reply:
Another thing that I was thinking, is that we might be able to graph the available responders as a forecast. So for example, we show current active responders and then we show active responders 30 minutes an hour and an hour and a half from now